ansible_base/authentication/models/authenticator_map.py

```diff
@@ -35,38 +36,49 @@ class Meta:
         default=False,
         help_text=(_("If a user does not meet this rule should we revoke the permission")),
     )
+    map_type_choices = [
+        ('allow', 'allow'),
+        ('is_superuser', 'is_superuser'),
+        ('organization', 'organization'),
+        ('team', 'team'),
+    ]
+    if 'ansible_base.rbac' in settings.INSTALLED_APPS:
```

Comment:
TODO: If rbac dependency is not optional, let's remove the condition

Reply:
I don't think our implementation of authentication_maps can exist without RBAC.

Reply:
This condition would conflict with the migration. Allowing using with/without the other app would be inflexible, at best.